src/main/java/com/enterprise/eventmanagementsystem/service/EventService.java

```diff
@@ -42,15 +44,17 @@ public Event save(Event event) throws Exception {
     }
 
     @Override
+    @Cacheable(value = "event", key = "#id")
     public Event fetch(int id) {
         return eventDAO.fetch(id);
     }
 
     @Override
-    @CacheEvict(value = "delete", key = "#id")
+    @CacheEvict(value = "event", key = "#id")
     public void delete(int id) {
         this.eventRepository.deleteById(id);
     }
+
```

Comment:
Adding caching as well as changed the naming of the caches as this was introduced as good coding practice in the class's video about the implementation of caching.